libs/common-services/src/main/java/uk/gov/di/ipv/core/library/domain/CredentialEvidenceItem.java

```diff
@@ -1,10 +1,10 @@
-package uk.gov.di.ipv.core.library.gpg45.domain;
```

Comment:
you moved this from gpg45-evaluator service....why?

Reply:
verifiable-credentials' VerifiableCredentialJwtValidator & gpg45-evaluator depend on CredentialEvidenceItem, but if it's in gpg45-evaluator then verifiable-credentials is dependent on gpg45-evaluator. This does work when a library gpg45-evaluator depends on also depends on verifiable-credentials because it creates a circular dependency. So I thought, to allow the cimit-services refactor I'd put this class (and it's two class dependencies) into common-services